examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out

```diff
@@ -1,13 +1,7 @@
-# Create a Ballerina project and a module inside it.
-# Generate mock service and stub code using `grpc_bidirectional_streaming_cont.proto` file.
-# For example, if you create a module named 'service', execute the following command to generate Ballerina code.
-$ ballerina grpc --input grpc_bidirectional_streaming_cont.proto --output src/service --mode service
+# Create new Protocol Buffers definition file `grpc_bidirectional_streaming.proto` and add service definition.
+# Run the following command in Ballerina tools distribution for stub generation.
+$ ballerina grpc --input grpc_bidirectional_streaming.proto  --output stubs
 
-# Once you run the command, `grpc_bidirectional_streaming_cont_pb.bal` and `Chat_sample_service.bal` files are generated inside service module.
-# Replace the content of `Chat_sample_service.bal` file with the content of `grpc_bidirectional_streaming.bal` file.
+# Once you run the command, `grpc_bidirectional_streaming_pb.bal` file is generated inside stubs directory.
 
-# Execute the following command to build the 'service' module.
-$ ballerina build service
-
-# Run the service using the following command.
-$ ballerina run target/bin/service-executable.jar
+# Please refer example `Proto To Ballerina` to get information on how to Ballerina Protocol Buffers tool.
```

Comment:
Typo:
"Please refer example `Proto To Ballerina` to get information on how to **use** Ballerina Protocol Buffers tool."

Reply:
Fixed all examples in commit 226d7c8928c4fc7893cc6b34d9920c0fb05d26ba